assets/js/components/dashboard/dashboard-app.js

```diff
@@ -39,6 +43,26 @@ class DashboardApp extends Component {
 				<div className="googlesitekit-module-page">
 					<div className="mdc-layout-grid">
 						<div className="mdc-layout-grid__inner">
+							<div className="
+								mdc-layout-grid__cell
+								mdc-layout-grid__cell--span-10-desktop
+								mdc-layout-grid__cell--span-6-tablet
+								mdc-layout-grid__cell--span-2-phone
+							">
+								<PageHeader
+									title={ __( 'Site Overview', 'google-site-kit' ) }
```

Comment:
Now that you removed the class from here, the title renders with a class `googlesitekit-heading-3` which makes it render small. Sorry, I didn't see before that that value was the default in `PageHeader` (I have no idea why it is), and now that the class name was removed, it becomes active. Let's make this in line with how the dashboard details screen renders its page header (https://github.com/google/site-kit-wp/blob/develop/assets/js/components/dashboard-details/dashboard-details-app.js#L61):
* Add class `googlesitekit-heading-2`.
* Add another class `googlesitekit-dashboard__heading`.

For consistency, wrap the above div `.mdc-layout-grid` in another div with class `googlesitekit-dashboard` (similar to https://github.com/google/site-kit-wp/blob/develop/assets/js/components/dashboard-details/dashboard-details-app.js#L49). This keeps the BEM tree consistent (even if we right now don't apply specific styles to that class).

Reply:
Ah, yes, I forgot about that myself—I noticed it happening when I was first adjusting this file; the API for `PageHeader` is a bit unexpected. 😓

The BEM styling `<div>` is quite confusing to me; I'm curious why we would add a div to a page for no reason. I would expect every div on a page to semantic organise elements or have a clear styling purpose, but in this case it seems only there for developer consistency. I suppose there's a future consideration there, but I worry about relying on div structure like this because it's very difficult to know/internalise/review. I guess that's a comment for another time, but intricate div structures like that deserve their own React components that will prevent errors like this one from happening in the future. 😄

Reply:
I completely agree with you, there's obviously still a need for a lot of refactoring, including revisiting this. For now, we should value making small improvements while also focusing on consistency. E.g. if we improved something like that here, but not in other components, it might make the code even harder to understand in the future.
I prefer your approach of discussing this another time, but on a large-scale level where we'll do it properly everywhere it's currently flawed.